Texture revisit

JorisDeBeer opened the pull request:
### Texture and palette overhaul

- Recreates palettes for better and more consistent looks
- New gem textures
- Modifies some template textures to make them closer to vanilla

Closes #330 closes #461

JorisDeBeer committed:
Test

JorisDeBeer committed:
Test

JorisDeBeer committed:
Gem textures/palettes
Tools edited to make place for gems
Better handle texture
Added Quartz gem textures

JorisDeBeer committed:
Polar bear pelt is now the same as vanilla

JorisDeBeer committed:
New Diamond Palette

JorisDeBeer committed:
Refitted the axe reinforcement to fit the new axe model.
New axe binding

JorisDeBeer committed:
Palette updates for the full vanilla line up.
Most palettes are simplified with alot less colors that where not used.
Cleaned up textures to make them fit vanilla standards.
Added mangrove

JorisDeBeer committed:
Fixed shovel head tip reinforcement to fit with new shovel texture

JorisDeBeer committed:
Delete some test stuff I forgot

JorisDeBeer committed:
Delete some test stuff I forgot

SigmundGranaas commented:
Some general feedback on the palettes and textures:
Overall, looks great and goes a long way towards making the texture feel A LOT closer to the vanilla feeling, so great work!

There are a couple of mostly palette things I want to touch on:
Generally, all vanilla items have a darker outline to make them stand out better from the background, this seems to be somewhat missing on some palettes:
![image](https://user-images.githubusercontent.com/46557903/220566709-9d1429fe-7bcd-4d44-b1ed-97e5314adec4.png)

I really like the new colors here, but we should add 1 or 2 distinctly darker pixels to the start of the palette to create the outline.
This seems to be the case for Andesite, Diorite, calcite, (Maybe deepslate?)
Before:
![image](https://user-images.githubusercontent.com/46557903/220592465-09449cad-3181-489d-a050-b18007d910e5.png)
Some tweaking after:
![image](https://user-images.githubusercontent.com/46557903/220593902-3d619797-c395-4d26-abde-9e53aeaea8c1.png)



Copper, crimson and warped needs just a little bump to make the outline a bit darker, but overall the colors look really nice and is definitely an improvement.
Before: 
![image](https://user-images.githubusercontent.com/46557903/220591053-8d950eab-ed42-46c9-817b-d8242f09467e.png)

After:
![image](https://user-images.githubusercontent.com/46557903/220591925-745f4a34-77cf-459e-a319-9a947acd0968.png)


I like the new overall color palette for the netherite, but currently the palette is only 5 pixels, and I think we can extend it by 2-3 pixel to better fill out some of the available pixels in the template:

![image](https://user-images.githubusercontent.com/46557903/220568499-7b9f4b5c-980f-4ece-be31-a069dcdb71e1.png)

Extended palette:
![image](https://user-images.githubusercontent.com/46557903/220571222-b62aa1a0-1c83-4b79-8865-086544397207.png)

Diamond palette is also slighty shortened, causing the system to ignore the second-darkest template outline:
![image](https://user-images.githubusercontent.com/46557903/220574230-26ff95e3-4621-4b1b-a033-2b233302eb5f.png)

Extended palette:
![image](https://user-images.githubusercontent.com/46557903/220575187-e39a0d15-f583-45cb-b1cf-f1c95f73d305.png)
Overview:
![image](https://user-images.githubusercontent.com/46557903/220575381-96935123-9aa5-490c-b751-0bb2239b1ea5.png)


There's also some issues with the template texture for a pickaxe head variant and the sword binding:
![image](https://user-images.githubusercontent.com/46557903/220576107-38321c67-ed29-447d-b6a9-dcd427bb3ee2.png)

This seems largely to be an issue with color palettes not being big enough to cover all of the different shades in the template. I think material palettes needs at least 6-7 colors to properly cover the templates as intended.

Here's a slightly tweaked template with an extended iron palette:
![image](https://user-images.githubusercontent.com/46557903/220590606-1df40434-5efc-4a07-916b-0c278c422ebc.png)

JorisDeBeer committed:
Merge branch 'SigmundGranaas:1.19' into Texture_revisit

SigmundGranaas committed:
Tweaked textures

SigmundGranaas committed:
Merge remote-tracking branch 'origin/Texture_revisit' into Texture_revisit

JorisDeBeer committed:
Deleted a lot of double files that were not needed (again)
Schematics and materials (only the vanilla ones right now) are redone and made to fit the new 7 color palette standard.
Revisited alot of schematics along the way.

JorisDeBeer committed:
New binding
Pummel has a darker color in the middle to fit the vanilla style
Pickaxe head order changed a bit to let the reinforcement actually show?

JorisDeBeer committed:
reverted two files

JorisDeBeer committed:
Merge branch 'SigmundGranaas:1.19' into Texture_revisit
